config/urls.py

```diff
@@ -32,6 +32,8 @@
 
 urlpatterns = [
     path("admin/", admin.site.urls),
+    path("user/", include("user.urls")),
+    path("auth/", include("userauth.urls")),
```

Comment:
Check this article for url naming conventions.

https://restfulapi.net/resource-naming/